tikv-client/src/main/java/com/pingcap/tikv/PDClient.java

```diff
@@ -545,19 +557,42 @@ protected PDStub getAsyncStub() {
 
   private void initCluster() {
     GetMembersResponse resp = null;
-    List<URI> pdAddrs = getConf().getPdAddrs();
+    List<URI> pdAddrs = new ArrayList<>(getConf().getPdAddrs());
+    // shuffle PD addresses so that clients call getMembers from different PD
```

Comment:
What's the reason for different PD?

Reply:
`getConf().getPdAddrs()` is a list that came from configurations created by users. `pdAddrs` is a new list used to be shuffled. Because we always use the first address in the list to get the PD leader in init or retry. Shuffling is helpful to make loading balance.